### ISSUE TITLE
CTD, no matter if it's StyleSwitcher or Vanilla

### ISSUE THREAD

**Issue by mirodno** [muhopensores/dmc3-snd-drv#1]
Windows 11, latest steam version (what else could it be?), no luck. Tried updating SDL2 from official SDL repository on github, still no luck. What could possibly go wrong? No logs, no dumps, just silent crash. Restoring original snd.drv fixes the bug, but it's definitely not intended behaviour of this patch.

Or at least tell me how to call that scary black output window to trace the origin of problem. If it's on my side only, i'll close the issue.

**Comment by mirodno** [comment]
I don't know how, but during some random reinstallation issue got fixed! Maybe cause is SDL2 file, i tried without it — it finally launched, but without music. I placed new SDL2 from libsdl's github, and IT'S FINALLY FR%AKING WORKING!
Now I can enjoy DA BEST PC version of Devil May Cry 3 Special Edition with fixed music, thanks.

**Comment by muhopensores** [comment]
Weird, maybe depending sdl2 wasnt such a good idea in retrospect. iirc i had some issues with audio crackling in different sdl2 builds, or loop points acting weird i dont remember exactly

**Comment by ProjectXsent** [comment]
It also happens on my end for some reason.

Tested on another PC, it runs fine but when tested on my laptop it crashes. Probably seems related to sound drivers like Nahimic/A-Volute which I have on my MSI laptop.

Update: Game doesn't crash anymore.

I managed to remove Nahimic/A-Volute drivers and also proceeded to uninstall the default Realtek Audio driver and replace it with [pal1000's Realtek Audio Driver](https://github.com/pal1000/Realtek-UAD-generic).
